.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,31 @@
+name: 'Run tests'
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.10"
+
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
+
+      - name: Install dependencies
+        run: poetry install
+
+      - name: Create DB image
+        run: docker-compose -f docker-compose-test.yml up -d
```

Comment:
I think, this is already done in `docker-compose-test.yml` file?

Reply:
Without this tests can't run, it gives error that connection to database is refused.